code/modules/mob/living/silicon/pai/pai.dm

```diff
@@ -9,7 +9,7 @@
 	var/obj/machinery/camera/current = null
 
 	var/ram = 100	// Used as currency to purchase different abilities
-	var/list/software = list()
+	var/list/software = list("crew manifest","digital messenger")

```

Comment:
Would break consistency, software uses strings everywhere else; it would actually obfuscate the code to add defines here